pkg/sippyserver/dbloader_test.go

```diff
@@ -0,0 +1,212 @@
+package sippyserver
+
+import (
+	"fmt"
+	"reflect"
+	"sort"
+	"testing"
+
+	"github.com/andygrunwald/go-jira"
+	"github.com/openshift/sippy/pkg/db/models"
+)
+
+func findIssuesForVariants() (map[string][]jira.Issue, error) {
+	result := map[string][]jira.Issue{
+		"variants=azure,ovn": []jira.Issue{
+			{
+				ID: "1",
+				Fields: &jira.IssueFields{
+					AffectsVersions: []*jira.AffectsVersion{
+						{Name: "4.12"},
```

Comment:
With above comment in mind, maybe change this to 4.12.z and make sure we still get our match.